student-work/miriam-rosenberg/readme.md

```diff
@@ -1 +1,66 @@
-# Hello! My name is Miriam Rosenberg.
+# Miriam Rosenberg 
+<img src="https://media-exp1.licdn.com/dms/image/C4D03AQHVcmWaY0dFGA/profile-displayphoto-shrink_400_400/0/1641184275552?e=1649894400&v=beta&t=XNRA55zLvZMQEb1WDnVmqsMB_SC672IeLOsnv5fA1Ls" width="100" height="100">
+
+**Technical Writer in Training at [Our Best Words](https://ourbestwords.com/)**  
+• techcomm • documentation • localization • UXdesign
+
+## About
+
+First and foremost, I love it when things work simply and elegantly. It gives me great pleasure to solve problems, increase usability, and make the process smooth from the beginning to the end.
+I've always been practical, and ever since I was little, I have been fascinated with technology and gadgets. I liked when toys worked, and once I even fixed the knee of my Barbie doll with a fork. 
+Very early on, I realised that things only work when information and communication work hand in hand. In my search for the best methods of communication, I developed a passion for languages. As a result, I am always learning one. So far, I have mastered three: Polish, English, and Spanish.
+
+Knowing a language is not enough; you need to know how to use it. So I took up the opportunity to learn it through a Technical Writing course. 
+It opened my eyes wider than I expected and challenged my intellect. 
+Here is what I learnt: 
+1. The success of your product depends on the users knowing how to use it. 
+2. I step in to help with point 1. 
+3. The result is concise and straightforward documentation, an online Help system, or a User Guide.
+
+The best part of being a Technical Writer is that the intellectual journey never ends.
+
+## Experience
+
+**Technical Writer in Training at [Our Best Words](https://ourbestwords.com/)**  
+Mar 2021-Present  
```

Comment:
```suggestion
Mar 2021 &ndash; Present  
```

Reply:
Use this in all dates. While the dash is fine, this is cleaner code as it will never create an issue where the compiler doesn't understand.